numpy/random/mtrand/Python.pxi

```diff
@@ -16,7 +16,7 @@ cdef extern from "Python.h":
 
     # Float API
     double PyFloat_AsDouble(object ob) except? -1.0
-    long PyInt_AsLong(object ob)
+    long PyInt_AsLong(object ob) except? -1
```

Comment:
`PyInt_AsLong` may have thrown an exception if it returns -1
https://docs.python.org/2/c-api/int.html#c.PyInt_AsLong